cloud/pkg/cloudstream/config/config_test.go

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2020 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"encoding/pem"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/kubeedge/kubeedge/pkg/apis/componentconfig/cloudcore/v1alpha1"
+)
+
+var pemData = `verify return:0
```

Comment:
The pemData is too complex, is this necessary?

Reply:
I tried to make it simpler, but it didn't work, the `Decode` of the process is quite complicated to me, and I'd be thankful if you could show a simple example or give some tips.

Reply:
updated